src/commands/diff.rs

```diff
@@ -1,5 +1,5 @@
 pub async fn diff(config: &fpm::Config, files: Option<Vec<String>>, all: bool) -> fpm::Result<()> {
-    let snapshots = fpm::snapshot::get_latest_snapshots(config).await?;
+    let snapshots = fpm::snapshot::get_latest_snapshots(&config.root).await?;
```

Comment:
Not fond of this. Pass the `config` instead of individual fields.

Reply:
There was a reason to do this, `get_latest_snapshot()` can be used for either the current package or for the original package if this is a translation.